client/src/components/CardContainer/CardContainer.js

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import {Link} from "react-router-dom";
+import {Swiper, SwiperSlide} from "swiper/react";
+import {Navigation} from 'swiper'
+import {Typography, styled} from "@mui/material";
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import {CardItem} from '../index'
+
+
+import 'swiper/swiper.min.css'
+
+
+const cards = [
+    {
+        name: "Kyiv city walk",
+        region: "Kyiv region",
+        id: 1,
+        url: ""
+    },
+    {
+        name: "Lviv city walk",
+        region: "Lviv region",
+        id: 2,
+    },
+    {
+        name: "Donetsk city walk",
+        region: "Donetsk region",
+        id: 3,
+    },
+    {
+        name: "Odessa city walk",
+        region: "Odessa region",
+        id: 4,
+    },
+    {
+        name: "Dnepr city walk",
+        region: "Dnepr region",
+        id: 5,
+    },
+    {
+        name: "Chernivtsi city walk",
+        region: "Chernivtsi region",
+        id: 6,
+    },
+    {
+        name: "Ternopil city walk",
+        region: "Ternopil region",
+        id: 7,
+    },
+    {
+        name: "Lutsk city walk",
+        region: "Lutsk region",
+        id: 8,
+    },
+]
+
+const CardContainerWrapper = styled('div')({
+    maxWidth: "1150px",
+    margin: "0 auto",
+    padding: "0 20px",
+})
+
+const CardContainerHeaderContainer = styled('div')({
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+})
+
+const CardContainerHeader = styled(Typography)(({theme}) => ({
+    fontSize: "24px",
+    color: theme.palette.primary.main,
+    marginBottom: "0",
+}))
+
+
+const CardContainerAllTours = styled(Typography)({
+    fontSize: "12px",
+    color: "#282828",
```

Comment:
I think you do not need to use here font-size and color. They will be automatically taken from our theme. 
Also, I assume that this element has to be a Link as it should move as to all hot deals.

Please, fix it.